app/services/merchant_service.rb

```diff
@@ -0,0 +1,17 @@
+class MerchantService
+
+  def self.get_all_merchants
+    response =  Faraday.get('http://localhost:3000/api/v1/merchants')
```

Comment:
One small refactor option would be to have a method that establishes the connection to the API with Faraday and then for each method you'd only pass the endpoint's specific path. It's totally optional but can save a little time in the long run when working with bigger or more complex apps.